app/messaging/process-payment-message.js

```diff
@@ -12,15 +12,15 @@ async function processPaymentMessage (message, receiver) {
     const originalPaymentRequest = JSON.parse(JSON.stringify(paymentRequest))
     await enrichPaymentRequest(paymentRequest)
     await sendMessage(paymentRequest, ENRICHED)
-    await sendMessage({ paymentRequest, accepted: true }, ACCEPTED)
+    await sendMessage({ paymentRequest, accepted: true }, ACCEPTED, { to: paymentRequest.sourceSystem })
     await receiver.completeMessage(message)
     console.log('Payment request enriched:', util.inspect(paymentRequest, false, null, true))
     await sendEnrichmentEvent({ originalPaymentRequest, paymentRequest })
   } catch (err) {
     console.error('Unable to process payment request:', err.message)
     await sendEnrichmentErrorEvent(message.body, err)
```

Comment:
Just realised `message.body` is defined as `paymentRequest` above

Reply:
Is that an issue?  The message body should be a payment request on the inbound message.  On the response, the original payment request is only one property of it.

Reply:
No issue, just that we should pass `paymentRequest` into tht func rather than `message.body` for continuity